### ISSUE TITLE
Kubuntu 21.10 compilation issue

### ISSUE THREAD

**Issue by bartoldm** [blackadderkate/weather-widget-2#24]
Hello,
I'm using Plasma 5.23.5, I do have qt5-qmake and qt5-qmake-bin installed (versions 5.15.2+dfsg-12ubuntu1.1), and I'm getting the following error message:

-- No qmake Qt5 binary found. Can't check QT_INSTALL_PREFIX
CMake Error at /usr/share/ECM/modules/ECMQueryQmake.cmake:31 (message):
  No qmake Qt5 binary found.  Can't check QT_INSTALL_PLUGINS as required
Call Stack (most recent call first):
  /usr/share/ECM/kde-modules/KDEInstallDirs5.cmake:249 (query_qmake)
  /usr/share/ECM/kde-modules/KDEInstallDirs.cmake:15 (include)
  CMakeLists.txt:10 (include)

What could I do to make it build?
Best regards,
Bartek

**Comment by blackadderkate** [comment]
I'm still on Kubuntu 20.04LTS, so I'm having to rely on Google here.

I saw a bug report on Redhat Bugzilla containing one of your compilation error messages, which suggested installing qt5-qtbase-devel.

If that doesn't help I found a stackoverflow post that recommended installing QT Creator, as that should install literally everything required to compile QT applications. It's a bit extreme for my liking, but sounds like it should work...
